Cargo.toml

```diff
@@ -4,6 +4,7 @@ members = [
     "argon2",
     "balloon-hash",
     "bcrypt-pbkdf",
+    "fuzz",
```

Comment:
Exclude `fuzz` from the workspace, e.g. see the `benches` crate in the [stream cipher](https://github.com/RustCrypto/stream-ciphers) repository. It's not worth to pull the fuzz dependencies every time while testing the implementation crates. We can have a [separate CI job](https://github.com/RustCrypto/stream-ciphers/blob/master/.github/workflows/benches.yml) to check that `fuzz` compiles without errors.